index.html

```diff
@@ -88,7 +88,7 @@
               <label for="search"></label>
             </div>
           </div>
-          <div class="row">
+          <div class="row" id="selectAll">
```

Comment:
remove bootstrap js and jquery js from script tag. Use vanilla JS for functionality and materialize JS for designing